view/adminhtml/web/vue/MenuType/CustomUrl.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div class="admin__field field">
+        <simple-field
+            :label="config.translation.customURL"
+            id="custom_url"
+            type="text"
+            v-model="item.content"
+        >
+        </simple-field>
+        <div class="admin__field field field-title">
+            <label
+                class="label admin__field-label"
+                for="snowmenu_custom_target"
+            >
+                {{ config.translation.targetLabel }}
+            </label>
+            <div class="admin__field-control control">
+                <input
+                    class="checkbox"
+                    type="checkbox"
+                    id="snowmenu_custom_target"
+                    true-value="_blank"
+                    false-value="_self"
```

Comment:
How does it work? I don't see any reference to these props.

Reply:
```v-model="item.target"```

Reply:
Ah, it's a native Vue thing :)
https://vuejs.org/v2/guide/forms.html#Checkbox-1